Source/BusinessEntities/BusinessEntities/BusinessEntities.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Rename the csproj file to Blackjack.BusinessEntities for consistency with how we name our other csproj files